### ISSUE TITLE
Navigating by arrow keys

### ISSUE THREAD

**Issue by csholmq** [bripkens/project-sidebar#5]
After toggling the project sidebar, I expected to be able to switch between the projects using the arrow keys.

Is this possible to implement?

**Comment by bripkens** [comment]
Yes, this would be possible to implement. You are welcome to take a stab at it if you would like to!
